packages/presets/src/index.ts

```diff
@@ -1,3 +1,5 @@
+import '@blocksuite/affine-shared/commands';
```

Comment:
What's this used for?

Reply:
Apply global type augmentations of commands like:
```
declare global {
  namespace BlockSuite {
    interface Commands {
      copySelectedModels: typeof copySelectedModelsCommand;
    }
  }
}
```
Otherwise there will be some TypeScript Error like `Property 'copySelectedModels' does not exist on type 'Chain<InitCommandCtx & CommandKeyToData<never>>'` in package build stage.